cogboard-webapp/src/components/widgets/dialogFields/DisplayValueSelect.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import {FormControl, MenuItem} from '@material-ui/core';
+import DropdownField from '../../DropdownField';
+import styled from "@emotion/styled/macro";
+
+/**
+ * Displays dropdown field using passed array of objects with the following properties: display, value.
+ * Example: [
+ *  {
+ *      display: "name1"
+ *      value: "value1"
+ *  },
+ *  {
+ *      display: "name2"
+ *      value: "value2"
+ *  }
+ * ]
+ */
+const DisplayValueSelect = props => {
+
+    const StyledDropdown = styled(FormControl)`
```

Comment:
Styled returns the component and components can't be defined inside other components, because on every render (potentially very often) you create new component. And I would rather change the name to StyledFormControl to be more semantic.